tools/objinfo.ml

```diff
@@ -167,8 +167,12 @@ let print_cmx_infos (ui, crc) =
     if not !no_code then
       Format.printf "functions@ %a@.@."
         Export_info.print_functions export
-  | Flambda2 _ ->
-    failwith "Flambda2 objinfo support not yet merged"
+  | Flambda2 None ->
+    printf "This is an Flambda 2 object with no export info\n"
```

Comment:
```suggestion
    printf "Flambda 2 unit\n"
```

Reply:
I've fixed a modified version of this, I'd like to see explicitly that it doesn't have export info, to avoid confusion.